deluge/common.py

```diff
@@ -957,17 +963,16 @@ def setup_translations(setup_gettext=True, setup_pygtk=False):
 
             if windows_check():
                 import ctypes
-                libintl = ctypes.cdll.intl
+                libintl = ctypes.cdll.LoadLibrary('libintl-8.dll')
                 libintl.bindtextdomain(domain, translations_path.encode(sys.getfilesystemencoding()))
                 libintl.textdomain(domain)
                 libintl.bind_textdomain_codeset(domain, "UTF-8")
                 libintl.gettext.restype = ctypes.c_char_p
 
-            # Use glade for plugins that still uses it
-            import gtk
-            import gtk.glade
-            gtk.glade.bindtextdomain(domain, translations_path)
-            gtk.glade.textdomain(domain)
+            # Use glade for plugins that still use it
```

Comment:
i dont know about in linux but at least one plugin still translates idk how many of the defaults have translations. However when switching from one language to another then back again i get 

 File "C:\Python27\lib\site-packages\deluge-1.4.0-py2.7.egg\deluge\ui\gtkui\lis
view.py", line 554, in add_column
   if header == column_state.name:
nicodeWarning: Unicode equal comparison failed to convert both arguments to Uni
ode - interpreting them as being unequal

and deluge doesnt start till i clear appdata